proto/streamlit/proto/WidgetStates.proto

```diff
@@ -51,9 +51,12 @@ message WidgetState {
     ArrowTable arrow_value = 11;
     bytes bytes_value = 12;
     FileUploaderState file_uploader_state_value = 13;
-    // String value that resets itself to empty after the script has been run.
-    // This is used for the chat_input widget.
+
+    // DEPRECATED: StringTriggerValue proto message is deprecated and unused.
+    // The ChatInputValue proto message should be used instead.
     StringTriggerValue string_trigger_value = 14;
+    // ChatInputValue resets itself to empty after the script has been run.
+    // This is used for the chat_input widget.
     ChatInputValue chat_input_value = 15;
```

Comment:
I think we can start adding the `deprecated` keyword in proto as well. Ken recently added this here to our protos, so I think it should be fine to use: https://github.com/streamlit/streamlit/blob/cb3f64d2ab927104cde2b9c685c0dfa07c7471f0/proto/streamlit/proto/ForwardMsg.proto#L80